mfthomps/CyberCIEGE-port-to-Unity3d
# Merge Request for Issue/294 - multiple choice dialog

Closes #294 

To test:
- Run Training/Life With Macros scenario
- Enable antivirus updates on Terry's computer
- Answer multiple choice questions as they appear
- Make sure scenario can succeed or fail based on choices
